### ISSUE TITLE
Button to clear highlights from all ideas

### ISSUE THREAD

**Issue by samdec11** [stride-nyc/remote_retro#184]
As a facilitator,
I would like one button to deselect all highlighted ideas,
So that I don't have to click each idea to clear my selections.

__By Way Of Explanation:__ A facilitator can highlight ideas to bring them to the attention of everyone in the retro. After multiple ideas have been highlighted, the facilitator has to deselect each of them individually when they want to clear the highlights.

__Acceptance Criteria:__

- [ ] When the facilitator highlights 1 or more ideas, a "clear highlights" button appears
  - [ ] When no ideas are highlighted, this button does not appear

**Comment by vanderhoop** [comment]
@imaginista i want to pull you into this story. it seems to me it this feature has a high utility given that sean and I both thought of the need for this feature without even discussing it. My question is: what's the UI component that appears, and--more importantly--_where_ does it appear? 

Can you come up with a couple rough design options for this feature?

And a follow-up consideration! Let's say multiple ideas are highlighted, as they were in our retro today: if the facilitator wanted to "group" these ideas into a category, whatever component exposes the "clear all highlights" functionality could have a sibling component that allows the facilitator to "group highlighted ideas". Just an idea, but I want to put it in your head!

**Comment by imaginista** [comment]
My gut tells me that the best way to approach this is to guide focus by reducing visual overload. If we go the route of highlighting by color while keeping all idea entries at 100% opacity I'd bet a 3 dollar bill that user focus would continue to wander the screen. If our goal with this aspect of the experience is to truly _focus_ people's attention I think we should explore a more reductive approach. Maybe darkening the screen, with only key ideas selected and at full contrast. Just put something together for us to illustrate what i'm talking about. Check it:

![rr-highlight](https://user-images.githubusercontent.com/1500663/28733996-c01eb9b8-73ac-11e7-9051-805aa41e0455.jpg)

**Comment by vanderhoop** [comment]
@imaginista i agree that the mock above would result in greater focus from participants. and I'm not opposed a couple conditions result from this:

1. if the facilitator wants to highlight additional ideas, the facilitator has a harder time _seeing_ the ideas he or she might want to highlight due to the dark overlay.
2. there's still no way to clear highlights for all ideas. what mechanism does the facilitator have to clear all highlights? 

____

implementation note for self (and other devs): we could accomplish the effect @imaginista mocked up by a adding a semi-transparent overlay whenever _any_ idea is highlighted, and giving any and all  highlighted ideas a z-index _greater_ than the overlay's.

**Comment by vanderhoop** [comment]
@samdec11 see comments above

**Comment by vanderhoop** [comment]
Got an idea from the way Slack allows your to extend certain user actions: 

Notice the animated dropdown in the gif below:
___

![only_appears_when_we_hover_over_dehighlight_idea](https://user-images.githubusercontent.com/4386645/30708031-9922a796-9eba-11e7-86b8-138c2ddb0e43.gif)
___

Provided that the following conditions are met:
  - A) there are multiple ideas highlighted on the board, and
  - B) the facilitator is hovering over the de-highlight icon

...we would provide a dropdown that says "Delight All Highlighted Ideas", or something similar


[10:37] 
you’d then display the dropdown
